pkg/controller/migmigration/dvm.go

```diff
@@ -78,6 +80,28 @@ func (t *Task) getDirectVolumeMigration() (*migapi.DirectVolumeMigration, error)
 	return nil, nil
 }
 
+func (t *Task) getDirectVolumeMigrationProgressForDVM(dvm *migapi.DirectVolumeMigration) (*migapi.DirectVolumeMigrationProgress, error) {
+
+	dvmLabels := dvm.GetCorrelationLabels()
+	selector := labels.SelectorFromSet(dvmLabels)
+	dvmpList := migapi.DirectVolumeMigrationProgressList{}
+	err := t.Client.List(context.TODO(),
+		&k8sclient.ListOptions{
+			LabelSelector: selector,
+		},
+		&dvmpList)
+
+	if err != nil {
+		return nil, liberr.Wrap(err)
+	}
+
+	if len(dvmpList.Items) > 0 {
```

Comment:
This should be cleaning up all DVMs with the label selector.

Reply:
Done, updated the code.